### ISSUE TITLE
make "Automated Profile Creation for User Login" more configurable

### ISSUE THREAD

**Issue by msramalho** [webrecorder/browsertrix-crawler#449]
An automated profile creation feature is super useful but as I understand [the current implementation](https://github.com/webrecorder/browsertrix-crawler/blob/c3b98e5047ea219336883b0b1969da425fc43456/create-login-profile.js#L224-L236) is somewhat limited in terms of the expected login form names.

For instance, it does not work for Twitter as different field names are used, but also due to it being a 2 step process (for Twitter). 

I wonder if there's any current way to easily expand the automated profile creation feature, and if not where is the best place to do so? Is in the browsertrix-profiles repository? Any tips or suggestions regarding this.

**Comment by ikreymer** [comment]
Yes, the automated profile creation is a bit limited currently. The plan is likely to expand that with user-supplied behaviors, which can also be used to run crawls. We hope to have a way of doing that more easily soon.
